source/vibe/db/postgresql/package.d

```diff
@@ -107,18 +108,19 @@ class __Conn : dpq2.Connection
     {
         import vibe.core.core;
 
-        auto sock = this.socket();
-
-        sock.blocking = false;
+        scope sock = this.socket();
+        auto triggerMode = FileDescriptorEvent.Trigger.any;
+        debug version(Have_vibe_core) triggerMode = FileDescriptorEvent.Trigger.read;
```

Comment:
debug?

Reply:
Yes, it is neede only for debug build, with release build it will somehow works because asserts are eliminated

Reply:
It's better to write it with `else`

Reply:
And please add explanation comment (as 1 in below of this msg)

Reply:
Very strange

Reply:
It is not better with else (or I can see how it could be written with else)

Reply:
```D
version(Have_vibe_core)
{
  enum triggerMode = FileDescriptorEvent.Trigger.read;
}
else
{
  enum triggerMode = FileDescriptorEvent.Trigger.any;
}

Reply:
But this is without Debug, but you are right, debug is not needed anyway, so yes, this is better

Reply:
Here is already used duplicated Socket (`scope sock = this.socket();` line). I am not propose to duplicate it too - just duplicate `socket_t` wrapped inside of `sock`

Reply:
Sure